.github/files/required-review.yaml

```diff
@@ -73,6 +73,7 @@
   paths:
    - 'projects/plugins/boost/**'
    - 'projects/plugins/super-cache/**'
+   - 'projects/js-packages/image-guide/**'
```

Comment:
I believe you'll need to add a corresponding exclusion above for this to work right.
https://github.com/Automattic/jetpack/blob/775904a89c873657d58cc4ed70964f0209ba4878/.github/files/required-review.yaml#L30-L39